foundation/org.eclipse.persistence.core/src/main/java/org/eclipse/persistence/internal/databaseaccess/DatabaseCall.java

```diff
@@ -1181,6 +1181,9 @@ public void translate(AbstractRecord translationRow, AbstractRecord modifyRow, A
                         parametersValues.add(value);
                     }
                 } else if (parameterType == LITERAL) {
```

Comment:
Interesting. So the `parameterType` is DatasourceCall.LITERAL and not DatasourceCall.OUT_CURSOR?